docker/ravendb-ubuntu-arm/healthcheck.sh

```diff
@@ -0,0 +1,7 @@
+pgrep Raven.Server > /dev/null
```

Comment:
Since it's still Ubuntu I would not duplicate `run-raven.sh` nor `healthcheck.sh`. Maybe we could have 2 or 3 dockerfiles in `ravendb-ubuntu` dir for each arch, a `.dockerignore` file and change `build-ubuntu` and `publish-ubuntu` to have an `-arch` switch.